stock_move_line_auto_fill/models/auto_fill_operation_mixin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AutoFillOperationMixin(models.AbstractModel):
```

Comment:
@AungKoKoLin1997 Is there actually a good point to have this mixin model?  These method are only used in stock.move model IIRC.

Reply:
Yes. We don't need to use it. I will remove this model and just add functions in `stock.move`